app/assets/javascripts/member-facing/backbone/braintree_hosted_fields.js

```diff
@@ -13,9 +13,15 @@ const BraintreeHostedFields = Backbone.View.extend({
   braintreeSettings() {
     return {
       id: "hosted-fields",
-      onPaymentMethodReceived: this.paymentMethodReceived,
+      onPaymentMethodReceived: this.paymentMethodReceived.bind(this),
       onError: this.handleErrors.bind(this),
-      onReady: this.hideSpinner.bind(this),
+      onReady: (bt) => {
+        this.deviceData = bt.deviceData
+        this.hideSpinner()
+      },
+      dataCollector: {
+        kount: { environment: 'sandbox' }
```

Comment:
Surely this'll need to be an env var or something equivalent? Not `sandbox` on production, right?